EPAS reorg: Phase 2 Database Administration and Upgrading topic restructure

francoughlin opened the pull request:
## What Changed?

Breaking up the EDB Resource Manager topic into multiple subtopics

nidhibhammar commented on product_docs/docs/epas/15/database_administration/02_edb_loader/building_the_control_file.mdx:
```diff
@@ -0,0 +1,411 @@
+
+-   `FREEZE= { FALSE | TRUE }`
+
+    Set `FREEZE` to `TRUE` to copy the data with the rows `frozen`. A tuple guaranteed to be visible to all current and future transactions is marked as frozen to prevent transaction ID wraparound. For more information about frozen tuples, see the [PostgreSQL core documentation](<https://www.postgresql.org/docs/current/static/routine-vacuuming.html).
```
```suggestion
    Set `FREEZE` to `TRUE` to copy the data with the rows `frozen`. A tuple guaranteed to be visible to all current and future transactions is marked as frozen to prevent transaction ID wraparound. For more information about frozen tuples, see the [PostgreSQL core documentation](https://www.postgresql.org/docs/current/static/routine-vacuuming.html).
```

nidhibhammar commented on product_docs/docs/epas/15/database_administration/02_edb_loader/control_file_examples.mdx:
```diff
@@ -0,0 +1,484 @@
+---
+title: "EDB* Loader control file examples"
```
```suggestion
title: "EDB*Loader control file examples"
```

nidhibhammar commented on product_docs/docs/epas/15/database_administration/02_edb_loader/data_loading_methods.mdx:
```diff
@@ -0,0 +1,24 @@
+---
+title: "Overview of data loading methods"
```
This should be the parent topic and Conventional path loading and direct path loading should be the child topics under this one

nidhibhammar commented:
I think we can re-order the topics under EDB*Loader like this:
- EDB*Loader key concepts and compatability
- Using EDB*Loader
- Overview of data loading methods
- Building the EDB*Loader control file
   - Control file examples
- Invoking the EDB*Loader
   - Conventional path loading
   - Direct path loading
   - Parallel direct path loading
   - Remote loading

nidhibhammar commented on product_docs/docs/epas/15/database_administration/10_edb_resource_manager/creating_resource_groups.mdx:
```diff
@@ -0,0 +1,278 @@
+---
+title: "Creating resource groups"
```
Can we have a diff nav title here, as this topic discusses creating, altering, and dropping resource groups?

djw-m replied:
Working with resource groups

francoughlin replied:
Will do!

nidhibhammar reviewed (changes requested):
I have fixed and committed typos. Also, I have given comments for re-ordering the EDB*Loader topic.

drothery-edb commented:
You need to test for broken links. I did a quick check and the link checker picked up quite a few.

francoughlin commented:
> 

Great idea, @nidhibhammar, thanks! I will make those changes.

francoughlin commented:
> I have fixed and committed typos. Also, I have given comments for re-ordering the EDB*Loader topic.

Thank you!

francoughlin commented:
Incorporated @nidhibhammar's comments, and have checked for and fixed broken links (thanks for your help, @drothery-edb).

Review for review, again. :-)
